io_scene_nif/modules/animation/material_import.py

```diff
@@ -39,15 +39,13 @@
 
 from pyffi.formats.nif import NifFormat
 
+from io_scene_nif.modules.animation.animation_import import Animation
 from io_scene_nif.utility import nif_utils
 from io_scene_nif.utility.util_global import NifOp
 from io_scene_nif.utility.util_logging import NifLog
 
 
-class MaterialAnimation:
-
-    def __init__(self, parent):
-        self.animationhelper = parent
+class MaterialAnimation(Animation):
```

Comment:
Big kudos for inheritance! 👍 I had tried the same some time ago on my end, but ran into some cryptic errors that I was unable to debug lol

Reply:
Yeah i reduced inatance method dependency ao was easier. You had done most of the leg work anyways